Replace Society with Board and Club with Society in the Backend and Frontend

RahulGopathi opened the pull request:
- Replaced all the occurrences of society/societies with board/boards and club/clubs with society/societies
- Updated the POR titles of Boards and Societies
- Passed the President of Boards and made it displayed on the frontend
- Added the field in the Board Model  for the constitution link and Added a button on the Board pages for the constitution link
  
   ![Constitution Link](https://i.imgur.com/WFHe275.png)

DarshitJain04 commented:
> * Replaced all the occurrences of society/societies with board/boards and club/clubs with society/societies
> * Updated the POR titles of Boards and Societies
> * Passed the President of Boards and made it displayed on the frontend
> * Added the field in the Society Model  for constitution link and Added a button on the Society pages for the constitution link

In the last point are you talking about the _Link to resources_ button?

RahulGopathi commented:
> > * Replaced all the occurrences of society/societies with board/boards and club/clubs with society/societies
> > * Updated the POR titles of Boards and Societies
> > * Passed the President of Boards and made it displayed on the frontend
> > * Added the field in the Society Model  for constitution link and Added a button on the Society pages for the constitution link
> 
> In the last point are you talking about the _Link to resources_ button?

No, I Added the Constitution Button now. Somehow I missed this commit before ;-;

DarshitJain04 commented:
> > > * Replaced all the occurrences of society/societies with board/boards and club/clubs with society/societies
> > > * Updated the POR titles of Boards and Societies
> > > * Passed the President of Boards and made it displayed on the frontend
> > > * Added the field in the Society Model  for constitution link and Added a button on the Society pages for the constitution link
> > 
> > 
> > In the last point are you talking about the _Link to resources_ button?
> 
> No, I Added the Constitution Button now. Somehow I missed this commit before ;-;

We are good to go 🚀
